tests/pipeline/test_estimates.py

```diff
@@ -0,0 +1,332 @@
+        PREVIOUS_STANDARD_DEVIATION:
+            EarningsEstimates.previous_standard_deviation.latest,
+        NEXT_STANDARD_DEVIATION:
+            EarningsEstimates.next_standard_deviation.latest,
+        PREVIOUS_COUNT:
+            EarningsEstimates.previous_count.latest,
+        NEXT_COUNT:
+            EarningsEstimates.next_count.latest,
+        PREVIOUS_FISCAL_QUARTER:
+            EarningsEstimates.previous_fiscal_quarter.latest,
+        NEXT_FISCAL_QUARTER:
+            EarningsEstimates.next_fiscal_quarter.latest,
+        PREVIOUS_HIGH:
+            EarningsEstimates.previous_high.latest,
+        NEXT_HIGH:
+            EarningsEstimates.next_high.latest,
+        PREVIOUS_MEAN:
+            EarningsEstimates.previous_mean.latest,
+        NEXT_MEAN:
+            EarningsEstimates.next_mean.latest,
+        PREVIOUS_FISCAL_YEAR:
+            EarningsEstimates.previous_fiscal_year.latest,
+        NEXT_FISCAL_YEAR:
+            EarningsEstimates.next_fiscal_year.latest,
+        PREVIOUS_LOW:
+            EarningsEstimates.previous_low.latest,
+        NEXT_LOW:
+            EarningsEstimates.next_low.latest
+    }
+
+    @classmethod
+    def get_dataset(cls):
+        return {sid:
+                pd.concat([
+                    cls.base_cases[sid].rename(columns={
+                        'other_date': RELEASE_DATE_FIELD_NAME
+                    }),
+                    df
+                ], axis=1)
+                for sid, df in enumerate(earnings_estimates_cases)}
+
+    loader_type = EarningsEstimatesLoader
+
+    def setup(self, dates):
+        _expected_next_release_date = self.get_expected_next_event_dates(dates)
+
+        _expected_previous_release_date = \
+            self.get_expected_previous_event_dates(
+                dates
+            )
+
+        cols = {}
+        cols[PREVIOUS_RELEASE_DATE] = _expected_previous_release_date
+        cols[NEXT_RELEASE_DATE] = _expected_next_release_date
+        for field_name in field_name_to_expected_col:
+            cols[field_name] = self.get_sids_to_frames(
+                zip_with_floats, field_name_to_expected_col[field_name],
+                self.prev_date_intervals
+                if field_name.startswith(PREVIOUS_PREFIX)
+                else self.next_date_intervals,
+                dates
+            )
+        return cols
+
+
+class BlazeEarningsEstimatesLoaderTestCase(EarningsEstimatesLoaderTestCase):
+    loader_type = BlazeEarningsEstimatesLoader
+
+    def pipeline_event_loader_args(self, dates):
+        _, mapping = super(
+            BlazeEarningsEstimatesLoaderTestCase,
+            self,
+        ).pipeline_event_loader_args(dates)
+        frames = []
+        for sid, df in iteritems(mapping):
+            frame = df.copy()
+            frame[SID_FIELD_NAME] = sid
+            frames.append(frame)
+        return bz.data(pd.concat(frames).reset_index(drop=True)),
+
+
+class BlazeEarningsEstimatesLoaderNotInteractiveTestCase(
```

Comment:
It uses a non-interactive symbol & swaps resources in. For more details probably @llllllllll knows...